src/components/top_bar/top_bar.ts

```diff
@@ -292,6 +294,11 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
   currentFormatName = "automatic";
   fontSizes = fontSizes;
 
+  get dropdownStyle() {
+    const { height } = this.env.model.getters.getSheetViewDimension();
```

Comment:
I feel liks its a bit strange to have things in the topbar dedepding on the grid size but I guess it's fine for now.

Reply:
After discussing with the team, it did seemse more logical to give that as props, so the topbar don't have to know about the grid, but the `Spreadsheet` does. Added a commit in your branch, you can squash it with yours